### DISCUSSION TITLE
Homepage setup

### DISCUSSION THREAD

**Discussion by mike13pocztafm** [netbox-community/netbox#11680] [Q&A] [answered]
Is it possible to set different homepage for each user or group. Example...

group A:
https://netbox.com/tenancy/tenants/1/

group B:
https://netbox.com/tenancy/tenants/2/

**Comment by jeremystretch** [comment, marked as answer]
No, but one of the features planned for NetBox v3.5 is [custom dashboard widgets](https://github.com/netbox-community/netbox/issues/9416), which will enable users to configure their individual dashboards to meet their needs and preferences.
